api/package.json

```
@@ -150,9 +149,9 @@
     "preinstall": "npx check-engine",
     "scalingo-postbuild": "node scripts/generate-cron > cron.json",
     "start": "node index.js",
-    "start:watch": "nodemon index.js",
+    "start:watch": "node --watch index.js",
```

Actuellement les options de configuration de node sont limitées par rapport à nodemon mais pour reproduire la configuration de node précédemment utilisée il faudrait plutôt un script comme ci-dessous.

Comme il n'y a pas de fichier de configuration pour node watch on ne peut que passer des options, ça fait donc beaucoup d'option à passer.

À noter que contrairement au fichier `nodemon.json` original le script proposé ci-dessous : 
* ne contient pas de watch sur le répertoire `tests` car les tests ne devraient pas avoir d'influence sur le code runtime utilisé en production
* ne contient pas de watch sur le fichier `package.json` car installer les dépendances peut prendre du temps et nécessite un redémarrage une fois qu'elles sont toutes chargées, donc autant savoir que le cas de mise à jour des dependencies n'est pas couvert par les watch

```suggestion
    "start:watch": "node --watch-path=./.env --watch-path=./index.js --watch-path=./src --watch-path=./lib --watch-path=./db/knex-database-connection.js --watch-path=./translations index.js",
```